ecosystem/nft-metadata-crawler-parser/src/worker.rs

```diff
@@ -46,8 +49,10 @@ pub struct ParserConfig {
     pub database_url: String,
     pub cdn_prefix: String,
     pub ipfs_prefix: String,
-    pub max_file_size_bytes: u32,
-    pub image_quality: u8, // Quality up to 100
+    pub ipfs_auth: Option<String>,
```

Comment:
Feels weird that the config has to pass in `?token=<key>`, can we just accept the key and then format the URL in the code accordingly?